joomla/joomla-cms
# [4.4] Fix help icon not opening help screen

Pull Request for Issue #42818 .

### Summary of Changes
Help icon fails to open help screen when clicked on the question mark.

Thanks @dgrammatiko for the solution.

### Testing Instructions
Go to any screen which has a help screen, for example Content -> Articles.
Click on the text "Help" of the help toolbar button and you will see the appropriate help screen. However, if you click on the question mark to the left of the text you will get a much larger blank window with "about:blank" in the address bar.

Download the prebuilt package.
or 
Apply PR and run `npm run build:js`
